requirements.txt

```
@@ -1,8 +1,8 @@
-Sphinx==7.1.1
+Sphinx==7.2.3
 furo>=2022.6.4
 jinja2
 sphinx-lint==0.6.8
-sphinx-notfound-page
+sphinx-notfound-page>=1.0.0rc1
```

It's been released: https://github.com/readthedocs/sphinx-notfound-page/issues/219#issuecomment-1698990942

```suggestion
sphinx-notfound-page>=1.0.0
```